Makefile

```diff
@@ -167,10 +167,6 @@ TEST_MYSQL_HOST ?= mysql:3306
 TEST_MYSQL_DBNAME ?= testgitea
```

Comment:
It seems there is a discrepancy between this database name and the action which has `test` as the name. For whatever reason this has worked so far, but I'll try setting them the same now.

Reply:
Maybe it's because gitea/gorm execute a `CREATE DATABASE`, so it likely wouldn't matter which database is pre-generated.

Reply:
Tests still pass so the change works.